.github/workflows/docker.yml

```diff
@@ -0,0 +1,39 @@
+name: Docker build
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  determine_docker_org_and_tag:
+    runs-on: ubuntu-latest
+    outputs:
+      docker_organization: ${{ steps.docker-org-and-tag.outputs.docker_organization }}
+      docker_image_tag: develop-humble
+    steps:
+      - id: docker-org-and-tag
+        uses: usdot-fhwa-stol/actions/docker-org-and-tag@main
+
+  build:
+    needs: determine_docker_org_and_tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and Push v2x-ros-conversion Docker Image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          pull: true
+          tags: ${{ needs.determine_docker_org_and_tag.outputs.docker_organization }}/v2x-ros-conversion:${{ needs.determine_docker_org_and_tag.outputs.docker_image_tag }}
+          build-args: |
+            DOCKER_ORG=${{ needs.determine_docker_org_and_tag.outputs.docker_organization }}
+            DOCKER_TAG=develop-humble
```

Comment:
We still want the image here to be tagged develop, not develop-humble

Reply:
I updated tag to develop and since we are not pushing the images from docker.yml I added arg push:fail, We will be pushing only images from dockerhub.yml when PR's are merged to main branches